### ISSUE TITLE
Allow multiple extensions to be installed/uninstalled through the CLI

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#6368]
By the looks of the code this is partially implemented, not sure if it's just incomplete or a bug.

Expected syntax:

```bash
code --install-extension ext1 ext2
```

**Comment by joaomoreno** [comment]
Duplicate of #6730.

**Comment by joaomoreno** [comment]
At best, we could implement it as 

`code --install-extension "ext1 ext2"`

Since parameterised arguments only take one parameter.

**Comment by Tyriar** [comment]
We could make it a boolean flag and all other args are the extensions?

**Comment by Tyriar** [comment]
Reopening as we should be able to improve the experience over adding --install/uninstall-extension multiple times

**Comment by joaomoreno** [comment]
If so, what should this do?

```
--install-extension foo --uninstall-extension bar
```

Should it install the extensions `foo`, `--uninstall-extension` and `bar`?

**Comment by Tyriar** [comment]
I was thinking similar to how npm works: `npm install foo uninstall`. Requiring `--install-extension` multiple times is so verbose.

**Comment by joaomoreno** [comment]
Could work, but I will simply close this. We enabled this so people can automate installations, this is not the usual way to install an extension; the product UI is for that. If someone wants to install multiple extensions, they can still do it.

**Comment by Tyriar** [comment]
K sounds good, I have already done [the automation I wanted to do](https://github.com/Tyriar/dotfiles/blob/master/modules/vscode/install_extensions.sh), just waiting on https://github.com/Microsoft/vscode/pull/6366 so it doesn't error out
